docker build error

i found below error :

```bash

 => ERROR [builder 9/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -a -o manager main.go                                                                                        0.7s
------
 > [builder 9/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -a -o manager main.go:
#15 0.628 api/v1alpha1/infrablockspace_types.go:20:2: no required module provides package github.com/InfraBlockchain/infrablockspace-operator/internal/chain; to add it:
#15 0.628       go get github.com/InfraBlockchain/infrablockspace-operator/internal/chain
#15 0.628 controllers/infrablockspace_controller.go:24:2: no required module provides package github.com/InfraBlockchain/infrablockspace-operator/internal/util; to add it:
#15 0.628       go get github.com/InfraBlockchain/infrablockspace-operator/internal/util
------
executor failed running [/bin/sh -c CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go]: exit code: 1
make: *** [docker-build] Error 1

```

please resolve this

--- a/Dockerfile
+++ b/Dockerfile
@@ -15,7 +15,7 @@ RUN go mod download
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
-
+COPY pkg/ pkg/
 # Build
 # the GOARCH has not a default value to allow the binary be built according to the host where the command
 # was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO

--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= $(IMAGE_TAG_BASE):v$(VERSION)
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.26.0
 

